values.yaml

```diff
@@ -7,8 +7,8 @@ replicaCount: 1
 product: orchestrate
 
 image:
-  repository: consensys-docker-pegasys-orchestrate.bintray.io/orchestrate
-  tag: v2.1.1
+  repository: ""
```

Comment:
Having a default value for images removes the need to configure it on the values.